sky/global_user_state.py

```diff
@@ -263,31 +275,32 @@ def set_cluster_metadata(cluster_name: str, metadata: Dict[str, Any]) -> None:
         raise ValueError(f'Cluster {cluster_name} not found.')
 
 
-def get_cluster_user_identity(cluster_name: str) -> Optional[str]:
-    metadata = get_cluster_metadata(cluster_name)
-    if metadata is None:
-        return None
-    return metadata.get('user_identity')
+def get_owner_identity_for_cluster(cluster_name: str) -> Optional[str]:
```

Comment:
`ag` shows unused.

Reply:
Ahh, good catch! We directly get it from the `record` we get using `get_cluster`. Removed it. Thanks!